src/accelerate/state.py

```diff
@@ -877,7 +891,7 @@ def __init__(
                     raise ValueError(
                         "Using `fp8` precision requires `transformer_engine` or `MS-AMP` to be installed."
                     )
-                elif not check_fp8_capability():
+                elif torch.cuda.is_available() and not check_fp8_capability():
```

Comment:
Is this specific to CUDA devices or should it be done for all non-HPU devices?

Reply:
This is specific to cuda, we can add one that checks for gaudi architecture.
both `check_fp8_capability()` and the warning under are calling `torch.cuda.get_device_compatibility() `which fails on hpu.